workbench-core/base/src/aws/helpers/dynamoDB/dynamoDBService.ts

```diff
@@ -290,60 +310,44 @@ export default class DynamoDBService {
    * const dataFromUpdate = await updater.item({'newAttribute': {S: 'newValue'}}).execute();
    * ```
    */
-  public update(
-    key: { [key: string]: unknown },
-    params?: {
-      disableCreatedAt?: boolean;
-      disableUpdatedAt?: boolean;
-      item?: { [key: string]: unknown };
-      set?: string;
-      add?: string;
-      remove?: string | string[];
-      delete?: string;
-      names?: { [key: string]: string };
-      values?: { [key: string]: unknown };
-      return?: 'NONE' | 'ALL_OLD' | 'UPDATED_OLD' | 'ALL_NEW' | 'UPDATED_NEW';
-      metrics?: 'NONE' | 'SIZE';
-      capacity?: 'INDEXES' | 'TOTAL' | 'NONE';
-    }
-  ): Updater {
-    let updater = new Updater({ region: this._awsRegion }, this._tableName, marshall(key));
-    if (params) {
-      if (params.disableCreatedAt) {
+  public update(updateParams: UpdateParams): Updater {
+    let updater = new Updater({ region: this._awsRegion }, this._tableName, marshall(updateParams.key));
+    if (updateParams.params) {
```

Comment:
Could we do early return here? I have strong opinion on if pyramids:
```
if...
  if...
    if...
      if...     
```
vs
```
if !...
  return...
if !...
   return...
etc...
```

Reply:
Doing it this way only requires a single `return` statement. The levels of the `if`s potentially build on each other so the final `return updater` captures all of this.

Reply:
Agree on second level `if`, but `if (!updateParams.params) return updater` should reduce nesting